IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_EnvironmentalCondition/DocPropertySet.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_EnvironmentalCondition" UniqueId="f70c35a7-7534-a696-f015-8118733a96ab" ApplicableType="IfcFurniture/TECHNICALCABINET,IfcSwitchingDevice/SELECTORSWITCH,IfcElementAssembly/TRACTION_SWITCHING_ASSEMBLY,IfcSwitchingDevice/EMERGENCYSTOP,IfcProtectiveDevice/FUSEDISCONNECTOR,IfcElectricFlowTreatmentDevice/ELECTRONICFILTER,IfcActuator/ELECTRICACTUATOR,IfcElectricFlowStorageDevice/COMPENSATOR,IfcElectricFlowStorageDevice/RECHARGER,IfcDistributionElement" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
Do we need such a specific list for this or can we just apply it to IfcElement like Pset_EnvironmentalImpactIndicators?